src/advanced/node-management.md

```diff
@@ -482,7 +492,8 @@ the HTTP status of the response is set to 404.
 
 ##### Known Uncommitted Transaction
 
-Response JSON has same fields as [`SerializedTransaction`](#serializedtransaction)
+Response JSON has same fields as
+[`SerializedTransaction`](#serializedtransaction)
 plus `type` field with value equal to `"in-pool"`.
 
 ??? example "Response Example"
```

Comment:
AFAIK the format has changed, causing some confusion among the users (@dip56 , is that correct?). Could anyone provide up-to-date examples?

Reply:
This file is still under correction. It will be updated in the next couple of days